rtos/rtx/TARGET_CORTEX_A/RTX_Conf_CA.c

```diff
@@ -74,7 +74,7 @@
 //   <i> Defines stack size for main thread.
 //   <i> Default: 200
 #ifndef OS_MAINSTKSIZE
- #define OS_MAINSTKSIZE 2048
+ #define OS_MAINSTKSIZE 4096
```

Comment:
Thank you for your comments.

We updated the comments of stack size for main threads.